docs/3_setup_guide/Delft_Fiat/data_input.qmd

```diff
@@ -0,0 +1,139 @@
+
+::: {.callout-tip}
+You can also create your exposure data with the [**Floodadapt  model builder**](hydromt_fiat_gui).
+:::
+
+### Types of Data
+Because FIAT consists of two models, the exposure data can be supplied in either a vector format or in raster format
+
+#### Vector data
+FIAT can be used to assess flood damages to buildings, roads, utilities, and other types of structures of which data is available. These type of assets are often spatially represented with vector, or geometry data. Building a FIAT model with geometry exposure data requires two data types:
+
+- **Geometry file(s)**
+- **CSV file**
+
+#### Raster Data
+
+::: {.callout-important}
+ This functionality is in beta phase
+:::
+
+Raster files must be **pre-processed** before being ingested into the FIAT model. Raster projection and extent must be coherent among exposure rasters and hazard maps. For each object type (e.g. residential buildings, industrial assets) a raster file with the **max potential damage** per cell is required. A damage function must be assigned to each raster file as well, linking with the damage function name in the meta data to the name in the **vulnerability curves file**.
+
+## Vulnerability Data
+The **vulnerability** of an asset is determined by its building type (e.g. 'residential 1-story building') and the inundation depth, also refered to as water depth, during a flood event. 
+
+::: {.callout-tip}
+You can also create damage functions with the [**Floodadapt  model builder**](hydromt_fiat_gui).
+:::
+
+Different assests incur different degrees of damage at varying inundation levels. This **vulnerability** can be quantified via **flood depth-damage** functions, see for example @fig-damagefunction. The damage function relates the water depth to the maximum potential damage per asset and returns the damage fraction (a value between 0 and 1). The damage fraction is multiplied by the maximum potential damage to obtain a damage value. The value of the maximum potential damage differs per asset and must be specified in the **exposure data**.
+
+
+```{python}
+#| echo: false
+#| label: fig-damagefunction
+#| fig-cap: "Damage functions of different assets/structures. "
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+from pathlib import Path
+
+
+file_path = Path.cwd()
+
+file_path = Path(file_path.parents[1], "_static", "files", "vulnerability_curves.csv")
+file_path = file_path.resolve()
+data = df = pd.read_csv(file_path, comment='#')
+df = pd.DataFrame({
+    ('water depth'): data.iloc[:, 0],
+    ('STRUCT1'): data.iloc[:, 1],
+    ('STRUCT2'): data.iloc[:, 2],
+
+})
+
+labels=["STRUCT1","STRUCT2"]
+plt.plot(df['water depth'], df['STRUCT1'])
+plt.plot(df['water depth'], df['STRUCT2'])
+plt.xlabel('Water depth (m)')
+plt.ylabel('Fraction of maximum potential damage')
+plt.legend(labels)
+plt.gca().get_legend().set_title('')
+```
+
+
```

Comment:
I'm sorry, but I think everything after this point should be removed. This belongs to the Delft-FIAT documentation. Users of the Delft-FIAT model-builder do not need to know any of this. This section should just be describing what the required data is, but not the formatting, because that will all be taken care of for them. And this makes it look like they need to know this.